### PR TITLE
Populate accomplishments table with accomplishments

### PR DESCRIPTION
**Pull request by ravishende**
Also filters and sorts them by users' choice

### CODE REVIEW

File: src/accomplishments/accomplishments.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
+import AccomplishmentsStorage from "../backend-storage/accomplishments-api.js";
+
+/**
+ * Get all accomplishment Objects from storage
+ * @returns {Object[]} array of accomplishments objects retrieved from storage
```
Comment (shambhavi-20) [inline src/accomplishments/accomplishments.js:5]: Is it possible to have the return be "AccomplishmentObj[]"?

File: src/accomplishments/accomplishments.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+            "There are no accomplishments. Add them in daily Logs.";
+        main.appendChild(noContentMsg);
+    }
+}
+
+/**
+ * Sort an array of accomplishments objects by oldest or newest given an unsorted or sorted array
+ * This allows for quick reversing betweeen newest and oldest rather than fully resorting each time
+ * @param {Object[]} accomplishmentsObjArr array of accomplishments objects
+ * @param {Object} options contains byOldest (boolean) and alreadySortedByOldest(boolean)
```
Comment (shambhavi-20) [inline src/accomplishments/accomplishments.js:32]: Should it be AccomplishmentObj?

File: src/accomplishments/accomplishments.js [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+    if (!options.byOldest) {
+        allAccomplishments = allAccomplishments.reverse();
+    }
+    return allAccomplishments;
+}
+
+/**
+ * Filters an array of accomplishments objects by current year, current month, or no filtering
+ * @param {Object[]} accomplishmentsObjArr array of accomplishments objects
+ * @param {Object} options contains byCurrentMonth (Boolean) and byCurrentYear (Boolean)
```
Comment (shambhavi-20) [inline src/accomplishments/accomplishments.js:65]: Should include a throws tag

**Review by shambhavi-20 — APPROVED** [review]
Everything looks good! I was looking at the documentation for JSdocs to see if custom class should be tagged as Object or the class name. I think it is okay to use Object but it might be more informative to use AccomplishmentObj in the param tags.

**Comment by ravishende** [comment]
Thanks! I'll fix those then merge.

**Pull request merged**